### PR TITLE
[MIG][11.0] account analytic default account

### PR DESCRIPTION
**Pull request by i-vyshnevska**

### CODE REVIEW

**Review by p-tombez — APPROVED** [review]
LG

File: account_analytic_default_account/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (tbaden) [inline account_analytic_default_account/models/__init__.py]: no longer required. please remove it

File: account_analytic_default_account/views/account_analytic_default_account_view.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            <field name="model">account.analytic.default</field>
+            <field name="inherit_id" ref="account_analytic_default.view_account_analytic_default_form" />
+            <field name="arch" type="xml">
+                <field name="user_id" position="after">
+                    <field name="account_id" />
+                </field>
+            </field>
+        </record>
+
+        <record id="view_account_analytic_default_account_search" model="ir.ui.view">
```
Comment (tbaden) [inline account_analytic_default_account/views/account_analytic_default_account_view.xml]: optional:
use original id (view_account_analytic_default_form_search)

File: account_analytic_default_account/views/account_analytic_default_account_view.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            <field name="model">account.analytic.default</field>
+            <field name="inherit_id" ref="account_analytic_default.view_account_analytic_default_tree" />
+            <field name="arch" type="xml">
+                <field name="user_id" position="after">
+                    <field name="account_id" />
+                </field>
+            </field>
+        </record>
+
+        <record id="view_account_analytic_default_account_form" model="ir.ui.view">
```
Comment (tbaden) [inline account_analytic_default_account/views/account_analytic_default_account_view.xml]: optional:
use original id (view_account_analytic_default_form)

File: account_analytic_default_account/views/account_analytic_default_account_view.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Camptocamp SA
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+        <record id="view_account_analytic_default_account_tree" model="ir.ui.view">
```
Comment (tbaden) [inline account_analytic_default_account/views/account_analytic_default_account_view.xml]: optional:
use original id (view_account_analytic_default_tree)

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review:
just a minor detail and some optional changes if you like.

File: account_analytic_default_account/views/account_analytic_default_account_view.xml (hunk 4 above)
Comment (i-vyshnevska) [reply account_analytic_default_account/views/account_analytic_default_account_view.xml]: @tbaden  thanks for review, updated

**Comment by tbaden** [comment]
@i-vyshnevska  could you please fix these minor travis warnings:
************* Module account_analytic_default_account
account_analytic_default_account/readme/DESCRIPTION.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
account_analytic_default_account/readme/CONTRIBUTORS.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
account_analytic_default_account/readme/USAGE.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
************* Module account_analytic_default_account.tests.test_analytic_default_account
account_analytic_default_account/tests/test_analytic_default_account.py:76: [W0622(redefined-builtin), TestAnalyticDefaultAccount.create_invoice] Redefining built-in 'type'

**Review by fclementic2c — APPROVED** [review]
functional test ok

**Pull request merged**